### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by imkiva**

### CODE REVIEW

**Comment by imkiva** [comment]
bors merge

**Comment by imkiva** [comment]
bors merge

File: gradle/deps.properties [hunk 1]
```diff
@@ -1,3 +1,3 @@
 version.project=0.0.1-SNAPSHOT
-version.aya=0.19-SNAPSHOT
+version.aya=0.19
```
Comment (ice1000) [inline gradle/deps.properties:2]: Not 0.20?

Comment (imkiva) [reply gradle/deps.properties:2]: It needs guest0x0 to disable gradle metadata

**Pull request merged**